src/table/components/TableTotals.jsx

```diff
@@ -18,7 +19,8 @@ function TableTotals({ rootElement, tableData, theme, layout, keyboard }) {
         const cellCoord = [isTop ? 1 : rows.length + 1, columnIndex];
         return (
           <TotalsCell
```

Comment:
I prefer `StyledTotalsCell` to keep consistent as others to indicate that it is styled.

Reply:
It's the child of `StyledHeadRow`. We have the similar naming `TableCell` in TableHeadWrapper as the child of 'StyledHeadRow'. Don't you think it's more fitted or you still prefer it to be changed?

Reply:
`TableCell` in 'StyledHeadRow' is not styled, so `TableCell` is correct for me. I see that when a component is styled, we add the 'styled' prefix.